File: android/src/main/java/com/lightningkite/kwift/observables/binding/SelectDayView.binding.actual.kt
package com.lightningkite.kwift.observables.binding

import com.lightningkite.kwift.observables.*
import com.lightningkite.kwift.views.android.SelectDayView
import java.util.*

fun SelectDayView.bind(day: MutableObservableProperty<Date?>) {
    this.selected = day.transformed(
        read = { it?.let { Calendar.getInstance().apply { timeInMillis = it.time } } },
        write = { it?.time }
    )
}

File: ios/Kwift/com/lightningkite/kwift/observables/binding/SelectDayView.binding.actual.swift
//Stub file made with Kwift 2 (by Lightning Kite)
import Foundation


//--- SelectDayView.bind(MutableObservableProperty<Date?>)
public extension SelectDayView {
    func bind(_ day: MutableObservableProperty<Date?>) -> Void {
        self.selected = day
    }
    func bind(day: MutableObservableProperty<Date?>) -> Void {
        return bind(day)
    }
}
